api_data.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+import json
+from urllib import urlopen 
+
+def output_cve_id(): 
+	"""Takes as input, a package name, package version. 
+	Queries cve-search' dataset for any reported 
+	vulnerabilities of the requested package. If 
+	vulnerability exists, outputs cve-id(s).
+	"""
+	package_name = raw_input('Enter package name: ')
```

Comment:
Yes, I was developing it into, command line based args and flags. 
Will try to get it done in the next commit.